editor/edit-post/modes/visual-editor/test/inserter.js

```diff
@@ -36,8 +36,15 @@ describe( 'VisualEditorInserter', () => {
 	it( 'should insert frequently used blocks', () => {
 		const insertBlock = jest.fn();
 		const mostFrequentlyUsedBlocks = [ getBlockType( 'core/paragraph' ), getBlockType( 'core/image' ) ];
+		const user = {
+			data: {
+				capabilities: {
+					upload_files: true,
+				},
+			},
+		};
 		const wrapper = shallow(
-			<VisualEditorInserter insertBlock={ insertBlock } mostFrequentlyUsedBlocks={ mostFrequentlyUsedBlocks } />
+			<VisualEditorInserter insertBlock={ insertBlock } mostFrequentlyUsedBlocks={ mostFrequentlyUsedBlocks } user={ user } />
```

Comment:
This is a long line, and I think we might consider splitting it across multiple:

```jsx
<VisualEditorInserter
	insertBlock={ insertBlock }
	mostFrequentlyUsedBlocks={ mostFrequentlyUsedBlocks }
	user={ user }
/>
```